src/libcore/syncAccount.js

```diff
@@ -90,6 +96,20 @@ export async function syncCoreAccount({
   return account;
 }
 
+function hideEmptyTokenAccounts(tokenAccounts: TokenAccount[]): TokenAccount[] {
```

Comment:
if you don't mind, i would prefer this logic to live in the core account iteration https://github.com/LedgerHQ/ledger-live-common/blob/2ea7e1105cbd9e6fd65368ad4ba64e2faa4be554/src/families/ethereum/libcore-buildTokenAccounts.js#L76

you can notice there is a function called buildERC20TokenAccount, we could even `return null` here https://github.com/LedgerHQ/ledger-live-common/blob/2ea7e1105cbd9e6fd65368ad4ba64e2faa4be554/src/families/ethereum/libcore-buildTokenAccounts.js#L20-L22 to indicate that "we skip this account", which is then just about doing `if(account)accounts.push(account)` there

wdyt?

Reply:
I totally missed that part, I assumed the token accounts array was coming straight from the libcore binary 😅
I'm refactoring right away.